docs/src/existing_mnemonic.md

```diff
@@ -0,0 +1,31 @@
+# existing-mnemonic
+
+## Description
+Uses an existing BIP-39 mnemonic phrase for key generation.
+
+## Optional Arguments
+
+- **`--chain`**: The chain to use for generating the deposit data. Options are: 'mainnet', 'holesky', etc...
+
+- **`--mnemonic`**: The mnemonic you used to create withdrawal credentials.
```

Comment:
There should be a note about using this argument from the CLI as a potential security risk with the value being stored on disk by the shell and being accessible from any admin account and potentially other malicious actors on the same machine. The same goes for all other password and seed related flags.

Reply:
I added a warning message to the top of each command page with a warning icon for each sensitive argument.